backend/sozisel/lib/sozisel_web/schema.ex

```diff
@@ -22,6 +23,7 @@ defmodule SoziselWeb.Schema do
   query do
     import_fields(:hello_queries)
     import_fields(:user_queries)
+    import_fields(:session_templates_queries)
```

Comment:
dont use plural name `:session_template_queries` should be enough, same goes with other types as well